cmd/launcher/run_compactdb.go

```diff
@@ -21,7 +23,14 @@ func runCompactDb(args []string) error {
 	}
 
 	// relevel
-	logger := logutil.NewServerLogger(opts.Debug)
+	slogLevel := slog.LevelInfo
+	if opts.Debug {
+		slogLevel = slog.LevelDebug
+	}
+
+	slogger := multislogger.New(slog.NewTextHandler(os.Stdout, &slog.HandlerOptions{
+		Level: slogLevel,
```

Comment:
If you want to include the source, need to update to:
```suggestion
		Level: slogLevel,
		AddSource: true,
```

Reply:
Thank you! I updated this one and all the other subcommands I hadn't added this to